src/test/resources/cucumber/features/componentTests/delete-branch.feature

```diff
@@ -17,17 +17,39 @@ Feature: CxFlow should delete SAST project when corresponding GitHub branch is d
       | false  | true      | false |
       | false  | false     | false |
 
+
   Scenario Outline: Github triggers delete event both when branch and tag are deleted,
   but CxFlow will call SAST delete API only when branch is deleted
     Given GitHub repo name is "VB_3845"
     And GitHub trigger is "<trigger>"
     And the "test2" branch is "not protected" as determined by application.yml
     And a project "VB_3845-test2" "exists" in SAST
     When GitHub notifies cxFlow that a "test2" ref was deleted
-    Then CxFlow will "<call>" the SAST delete API for the "VB_3845-test2" project
+    Then CxFlow will <call> the SAST delete API for the "VB_3845-test2" project
     And no exception will be thrown
 
     Examples:
       | trigger | call  |
       | branch  | true  |
-      | tag     | false |
\ No newline at end of file
+      | tag     | false |
+
+
+  Scenario Outline: CxFlow deletes SAST project when a non-protected Azure branch is deleted
+    Given Azure repo name is "Delete-Branch-Repo"
+    And Azure branch is "Test"
+    And the "Test" branch is <protected> as determined by application.yml
+    And the Azure properties is <setToDelete> by application.yml
+    And config-as-code in azure default branch include Cx-project <configAsCodeProject>
```

Comment:
This scenario tests 2 things at once: 1) determining config-as-code branch 2) checking whether SAST 'delete project' API was called. These 2 items should be in separate scenarios.